frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -131,7 +131,10 @@ const EditReceiverSettingsForm: React.FC<EditReceiverSettingsFormProps> = ({
             setOrgReceiverSettingsNewJson(
                 JSON.stringify(orgReceiverSettings, jsonSortReplacer, 2)
             );
-            if (latestResponse?.version !== orgReceiverSettings?.version) {
+            if (
+                action === "edit" &&
```

Comment:
Only do the "did someone edit this while I was working on it" check if doing and edit operation (Not for clone)